modules/service/src/main/java/com/devonfw/module/service/common/impl/CorsConfigProperties.java

```diff
@@ -0,0 +1,83 @@
+package com.devonfw.module.service.common.impl;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+/**
+ *
+ *
+ */
+@ConfigurationProperties
+public class CorsConfigProperties {
+  private boolean allowCredentials;
+
+  private String allowedOrigin;
+
+  private String allowedHeader;
+
+  private String allowedMethods;
+
+  /**
+   * @return allowCredentials
```

Comment:
I actually do not know myself what it does. So as you can see JavaDoc would really help.
Also add fully-qualified `@see` tags to all these getters e.g.
```
@see org.springframework.web.cors.CorsConfiguration#isAllowCredentials()
```